src/Structure/ObjectItemTrait.php

```diff
@@ -51,6 +51,8 @@ public function addAdditionalPropertyName($name)
      */
     public function getAdditionalPropertyNames()
     {
+        if(is_null($this->__additionalPropertyNames))
```

Comment:
Let's improve performance a bit by removing function call and return empty array to comply with original result expectations:

```suggestion
        if (null === $this->__additionalPropertyNames) {
            return [];
        }
```